qiskit/ignis/verification/accreditation/qotp.py

```diff
@@ -263,6 +263,22 @@ def QOTP(circ, num, two_qubit_gate='cx', coupling_map=None, seed=None):
     return qotp_circs, qotp_postps
```

Comment:
Also needs to fix the documentation of QOTP, the return part looks bad, see:
https://qiskit.org/documentation/stubs/qiskit.ignis.verification.QOTP.html#qiskit.ignis.verification.QOTP